packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -215,16 +215,24 @@
       </div>
       <div class="row">
         <div class="field-wrapper">
-          <a class="btn social-btn" :href="uris.github">
-            Continue With GitHub
-          </a>
+          <input
```

Comment:
Wondering why these 2 are inputs and not buttons

Reply:
I have a habit to use an input type button instead of button tag if it does not require some specific content inside. do you have some thought about `input type button` vs `button`

Reply:
Ah, missed the `type="button"` part. Not against it, they're both quite similar although one is more verbose than the other. I think people used to prefer the `input` way for backward compatibility with IE if im not mistaken.